app/presenters/content_row_presenter.rb

```diff
@@ -0,0 +1,19 @@
+class ContentRowPresenter
```

Comment:
Could you please add a unit test for this class?
It is good to have a descriptive test to describe code like:

```ruby
  @document_type = data[:document_type].try(:tr, '_', ' ').try(:capitalize)
```